CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -804,9 +810,198 @@ private static void OnTimeoutEvent(object source, ElapsedEventArgs e)
+            /// <param name="playerBets">List of antes values to be added to the pot.</param>
+            /// /// <param name="playersActive">Importing a list of all player indecies, all players eligible for main pot</param>
+            public void InitializePot(int _ante, List<int> playerBets, List<int> playersActive)
+            {
+                Pots.Add(new Pot(PotType.MAIN, 0, playersActive));
+
+                if (playerBets.All(n => n == playerBets[0])) //if all bets match the ante
+                {
+                    AddToPot(_ante, playerBets, playersActive);
+                }
+                else //create side pots if any bets do not match the ante
+                {
+                    CreateSidePots(_ante, playerBets, playersActive);
+                }
+            }
+            /// <summary>
+            /// Adds a bets to the pot.
+            /// </summary>
+            /// <param name="_currentBet">The amount each player is contributing to the pot.</param>
+            /// <param name="numPlayers">The number of players contributing to the pot.</param>
+            /// <param name="playersActive">The positions of the players that are currently eligible for the pot</param>
+            public void AddToPot(int _currentBet, List<int> playerBets, List<int> playersActive)
+            {
+                if (playerBets.All(n => n == playerBets[0]))
+                {
+                    Pots.First(pot => pot.PotType == PotType.MAIN).Total += _currentBet * playerBets.Count();
+                }
+                else
+                {
+                    CreateSidePots(_currentBet, playerBets, playersActive);
+                }
+            }
+            /// <summary>
+            //Adds bets of players that were folded to the pot
+            /// </summary>
+            /// <param name="playersBets"> list of bets that were placed but folded due to a raise in that round</param>
+            public void AddFoldedBetsToPot(List<int> playerBets)
+            {
+                if (playerBets.Count() == 0)
+                {
+                    return;
+                }
+                else
+                {
+                    for (int bet = 0; bet < playerBets.Count(); bet++)
+                    {
+                        Pots[0].Total += playerBets[bet];
+                    }
+                }
+            }
+            /// <summary>
+            /// Creates side pots as needed at the end of a round of betting.
+            /// </summary>
+            /// <param name="_currentBet">The amount to add to the pot.</param>
+            /// <param name="playerBets">The list of bets from players still active after the round of betting.</param>
+            /// <param name="playersActive">The list of players eligible to win the main pot, all in players will be subtracted for side pot</param>
+            public void CreateSidePots(int _currentBet, List<int> playerBets, List<int> playersActive)
```

Comment:
```suggestion
            public void CreateSidePots(int currentBet, List<int> playerBets, List<int> playersActive)
```

remove the underscore in the parameter name.